app/controllers/api/messaging.php

```diff
@@ -1832,8 +1854,14 @@
             throw new Exception(Exception::SUBSCRIBER_NOT_FOUND);
         }
 
+        $user = Authorization::skip(fn () => $dbForProject->getDocument('users', $subscriber->getAttribute('userId')));
+
         $response
-            ->dynamic($subscriber, Response::MODEL_SUBSCRIBER);
+            ->dynamic(
+                $subscriber
+                    ->setAttribute('userName', $user->getAttribute('name')),
```

Comment:
Same here, let's do this before dynamic call

Reply:
fixed this